### DISCUSSION TITLE
How to Obtain Multiple Predictions for Short Single-Word Audio Using Whisper?

### DISCUSSION THREAD

**Discussion by GeorgeAkkarapong** [openai/whisper#1848] [Q&A]
How can I use Whisper to predict single-word utterances from an audio file and receive multiple alternative predictions for each input, along with corresponding confidence scores? For example, if the audio file contains the word 'cheap,' I want to receive multiple predictions like ['sheep', 'cheap', 'cheese'] with corresponding confidence scores like [0.91, 0.88, 0.57].

I acknowledge that a custom classification AI model could provide a solution, but I opt for Whisper due to its extensive vocabulary of 49,000 words in its pretrained large model.

**Comment by glangford** [comment]
FYI, past discussions:

* https://github.com/openai/whisper/discussions/284
* https://github.com/openai/whisper/discussions/1183
* https://github.com/openai/whisper/discussions/1619
